### PR TITLE
try .yml extension if .yaml does not exist

### PR DESCRIPTION
**Pull request by developer-guy**
## 🎯 What
Provide a concise description of the changes:
* ❓ What changes have you made? (High-level overview)
* 🎉 What does it mean to the user? (In plain English)

Example:
* 🚀 Upgraded Terraform AWS provider to v3.0.0.
* 🎉 This means users can now use new features and improvements from the latest AWS Provider.

## 🤔 Why
Explain why the changes are necessary:
* 💡 Why were these changes made?
* 🎯 What's the benefit?

Example:
* 💡 We made this upgrade to take advantage of performance improvements and new features in the latest version.
* 🎯 This will allow us to provision infrastructure more efficiently, and utilize new AWS services and features.

## 📚 References
Link any supporting context or documentation:
* 🔗 Github issues, documentation, helpful links.
* ✅ Use `closes #123`, if this PR closes a GitHub issue `#123`

### CODE REVIEW

**Comment by developer-guy** [comment]
kindly ping @Excoriate

**Comment by Excoriate** [comment]
Thanks @developer-guy ❤️ ! I'll take a look during today, and I'll be back with you as soon as I can.
